docs/source/cli_ref/usage/WORKFLOWS.rst

```diff
@@ -127,6 +127,16 @@ network with a tower-cli command after the constituent resources like
 the job templates and projects were created by preceding tower-cli
 commands.
 
+Workflows Inside Workflows
```

Comment:
This is good. How about adding `workflow` above as well in the example https://github.com/ansible/tower-cli/pull/750/files#diff-e888d13496218a3618bbee8be0d2a7d1R114-R124 ? I see example already shows jt, project, inv_source, so we can add `workflow` there and make it a complete example. Just a suggestion.

Reply:
I hope you don't mind that I don't do that out of sheer laziness. I _know_ that I've tested the content here is functional. The implication is that you can combine them all together, but we just don't want poke this library any more than we have to.